server/services/locationsService.test.ts

```diff
@@ -123,10 +123,20 @@ describe('Locations service', () => {
   })
 
   describe('getLocation', () => {
-    it('calls the correct client function', async () => {
+    it('calls the correct client function when not including history', async () => {
       await locationsService.getLocation('token', '481fc587-60f8-402b-804d-64462babddcc')
```

Comment:
We can use a constant like in this way.
const **locationId** = '481fc587-60f8-402b-804d-64462babddcc'
then we replicate that across the test/tests (including json, etc)
  await locationsService.getLocation('token', **locationId**)